pkgs/development/python-modules/beancount_docverif/default.nix

```diff
@@ -14,6 +15,10 @@ buildPythonPackage rec {
     sha256 = "1kjc0axrxpvm828lqq5m2ikq0ls8xksbmm7312zw867gdx56x5aj";
   };
 
+  buildInputs = [
```

Comment:
```suggestion
  nativeBuildInputs = [
```

Reply:
Agreed, this is cleaner. Respun and force-pushed.